roles/tackle/defaults/main.yml

```diff
@@ -167,10 +168,13 @@ windup_service_name: "{{ app_name }}-{{ windup_name }}-{{ windup_component_name
 windup_container_requests_cpu: "500m"
 windup_container_requests_memory: "2Gi"
 
-maven_name: "maven"
-maven_data_volume_size: "100Gi"
-maven_data_volume_name: "{{ app_name }}-{{ maven_name }}-data"
-maven_data_volume_claim_name: "{{ app_name }}-{{ maven_name }}-volume-claim"
+cache_name: "cache"
+cache_data_volume_size: "100Gi"
+cache_data_volume_name: "{{ app_name }}-{{ cache_name }}-data"
+cache_data_volume_claim_name: "{{ app_name }}-{{ cache_name }}-volume-claim"
+cache_data_volume_mode: "ReadWriteMany"
```

Comment:
perhaps cache_data_volume_claim_mode to make it consistent with rest of components

Reply:
sure!